SwiftVideoBackground/MoreViewController.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+class MoreViewController: UIViewController {
+    @IBOutlet weak var view1: UIView!
+    @IBOutlet weak var view2: UIView!
+
+    let videoBackground1 = VideoBackground()
+    let videoBackground2 = VideoBackground()
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.isNavigationBarHidden = false
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        navigationController?.isNavigationBarHidden = true
+    }
+
+    // Video 1 - loads from web so takes a moment to start
+    @IBAction func play1(_ sender: Any) {
+        let url = URL(string: "https://storage.googleapis.com/coverr-main/mp4/Mt_Baker.mp4")!
+        videoBackground1.play(view: view1, url: url, alpha: 0.1)
+    }
+
+    @IBAction func increaseAlpha(_ sender: Any) {
+        videoBackground1.alpha += 0.1
+    }
+
+    @IBAction func decreaseAlpha(_ sender: Any) {
+        videoBackground1.alpha -= 0.1
+    }
+
+    @IBAction func toggleRestart(_ sender: Any) {
+        videoBackground1.willLoopVideo = !videoBackground1.willLoopVideo
+    }
+
+    // Video 2
+    @IBAction func play2(_ sender: Any) {
+        try? videoBackground2.play(view: view2, name: "pokemon", type: "mp4", isMuted: false)
```

Comment:
I love this. (Michelle says she approves too). However, I worry... will there be any copyright issues with us using this video?

Reply:
I don’t know haha. It made testing so enjoyable...still one of the best intros of all time. Ill just find another random clip just in case. The main point was I needed a vid with sound in order to test those interactions